tests/gso.test.ts

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+
+import { expect } from '@jest/globals';
+import { clusterApiUrl } from '@solana/web3.js';
+import { GSO } from '../src';
+
+dotenv.config();
+
+describe('gso-sdk', () => {
+  const client = new GSO(
+    process.env.RPC_URL || clusterApiUrl('devnet'),
```

Comment:
default to mainnet since gso is stuck in governance on devnet, also because BONK on devnet isnt a thing

Reply:
If this blocks submission since we don't have any active gso, just switch back and leave a comment.